openreview/conference/matching.py

```diff
@@ -12,6 +12,11 @@ def clear(self, client, invitation):
         for note in note_list:
             client.delete_note(note)
 
+    def _jaccard_similarity(self, list1, list2):
```

Comment:
I think this calculation is correct. but it might be easier to read if you use the set `union` function instead of calculating it manually (also, you can call pass a set into `len()`, you don't need to convert to a list)

example:
```
set1 = set(list1)
set2 = set(list2)
intersection = set1.intersection(set2)
union = set1.union(set2)
return len(intersection) / len(union)
```

Reply:
also, python 3 doesn't require converting the final division to a float (division in python 3 does this automatically), but python 2.7 does. 

if you want to maintain backwards compatibility, you can either explicitly call `float()`, or you can add this import statement: `from __future__ import division`

Reply:
Thanks! I made both changes.